src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,36 @@
+package duke.tasks;
+
+public class Task {
+    protected String name;
+    protected boolean isDone;
+
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public boolean isDone() {
+        return this.isDone;
```

Comment:
You do not need to use the `this` keyword unless it is ambiguous (shadowed by method parameter like in `setName`)